custom_components/huesyncbox/strings.json

```diff
@@ -27,7 +29,6 @@
       "cannot_connect": "Unable to connect to the device",
       "already_configured": "Device is already configured",
       "already_in_progress": "Config flow for device is already in progress.",
```

Comment:
The Hassfest step in the Github action is complaining about the `strings.json`. I think it is the comma after this line. 
Just to be sure check the file with a validator.

Reply:
b3133a2